charts/stable/minecraft-java/values.yaml

```diff
@@ -85,7 +85,7 @@ service:
 mcbackup:
   enabled: true
   initial_delay: 2m
-  backup_interval: 24hr
+  backup_interval: 24h
```

Comment:
Can you please also adjust the typo in questions.yaml?
Thanks!

Reply:
Done.